__test__/components/LoginFormContainer.test.js

```diff
@@ -45,7 +44,6 @@ describe('test the login container form', () => {
     input1.simulate('change', { target: { value: 'fafa@gmail.com' } });
     input2.simulate('change', { target: { value: 'fafasecret33' } });
     expect(wrapper.state('error')).toEqual('error logging in');
-    expect(wrapper.find('.text-danger')).toHaveLength(5);
```

Comment:
Why was this removed please?

Reply:
This line was removed long ago (because it was failing). I noticed it was added back after Efe's branch was merged.